packages/manager/modules/manager-pci-common/src/components/rclone-download/StorageRegionSelector.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { useTranslatedMicroRegions } from '@ovh-ux/manager-react-components';
+import { OdsSelect } from '@ovhcloud/ods-components/react';
+import { TRegion } from '../../api/data';
+
+interface StorageRegionSelectorProps {
+  regions: TRegion[];
+  onRegionChange: (regionName: string) => void;
+}
+
+export function StorageRegionSelector({
+  regions,
+  onRegionChange,
+}: Readonly<StorageRegionSelectorProps>) {
+  const { translateMicroRegion } = useTranslatedMicroRegions();
+
+  return (
+    <OdsSelect
+      defaultValue={regions?.[0]?.name}
+      name="currentRegion"
+      data-testid="regions_select"
+      onOdsChange={(event) => {
```

Comment:
If I inspect the Event Type from ODS: 
```ts
interface OdsSelectChangeEventDetail {
    name: string;
    previousValue?: string | null;
    validity?: ValidityState;
    value: string | null;
}
```
There is no need IMO to stringify the `event.detail.value`

Reply:
done thanks